method/group.js

```diff
@@ -67,62 +75,63 @@ module.exports = function GroupMethods (ssb, keystore, state) {
     },
 
     addMember (groupId, authorIds, opts = {}, cb) {
-      const { writeKey, root } = keystore.group.get(groupId)
+      ssb.box2.getGroupInfo(groupId, (err, groupInfo) => {
+        if (err) return cb(err)
 
-      const content = {
-        type: 'group/add-member',
-        version: 'v1',
-        groupKey: writeKey.key.toString('base64'),
-        root,
-        tangles: {
-          members: {
-            root,
-            previous: [root] // TODO calculate previous for members tangle
+        const { writeKey, root } = groupInfo
+
+        const content = {
+          type: 'group/add-member',
+          version: 'v1',
+          groupKey: writeKey.key.toString('base64'),
+          root,
+          tangles: {
+            members: {
+              root,
+              previous: [root] // TODO calculate previous for members tangle
+            },
+
```

Comment:
```suggestion
            members: {  root, previous: [root]  },
```

I think the members tangle is done right

Reply:
done